### DISCUSSION TITLE
How to make a list in which your nickname is added when you click on the button?

### DISCUSSION THREAD

**Discussion by MZ-contain** [nextcord/nextcord#778] [Q&A]
I want to implement a list in an embed message.
In which, when pressed, the name of the person who pressed the button is added, and then mention him.
Hope you can help :)

**Comment by DenverCoder1** [comment]
Check this and the other View examples for starters

https://github.com/nextcord/nextcord/blob/master/examples/views/counter.py

To edit an Embed, you can get the embed from interaction.message with interaction.message.embeds[0] and edit its description, then use the edit_message response to edit the message with the new embed.

**Comment by MZ-contain** [comment]
Thanks for the answer, but alas, I did not find what I was looking for. Might be worth a visit :)
